docs/book/src/getting-started.md

```diff
@@ -0,0 +1,537 @@
+
+## Sample Project
+
+We will create a sample project to let you know how it works. This sample will:
+
+- Reconcile a Memcached CR
+- Create a Deployment with the Memcached image
+- Not allow more instances than the size defined in the CR which will be applied
+- Update the Memcached CR status 
+
+Following the steps.
+
+## Create a project
+
+First, create and navigate into a directory for your project. Then, initialize it using `kubebuilder`:
+
+```shell
+mkdir $GOPATH/memcached-operator
+cd $GOPATH/memcached-operator
+kubebuilder init --domain=example.com
+```
+
+## Create the Memcached API (CRD):
+
+Next, we'll create a new API responsible for deploying and managing our Memcached solution. In this instance, we will utilize the [Deploy Image Plugin][deploy-image] to get a comprehensive code implementation for our solution.
+
+```
+kubebuilder create api --group example.com --version v1alpha1 --kind Memcached --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,-o,modern,-v" --image-container-port="11211" --run-as-user="1001" --plugins="deploy-image/v1-alpha" --make=false
+```
+
+### Understanding APIs
+
+This command's primary aim is to produce the Custom Resource (CR) and Custom Resource Definition (CRD) for the Memcached Kind. It creates the API with the group cache.example.com and version v1alpha1, uniquely identifying the new CRD of the Memcached Kind. By leveraging the Kubebuilder tool, we can define our APIs and objects representing our solutions for these platforms. While we've added only one Kind of resource in this example, you can have as many Kinds as necessary. Simply put, think of CRDs as the definition of our custom Objects, while CRs are instances of them.
+
+<aside class="note">
+<h1>Getting a better idea</h1>
+
+Consider a typical scenario where the objective is to run an application and its database on a Kubernetes platform. In this context, one object might represent the App, while another denotes the DB. If we define one CRD for the App and another for the DB, we uphold essential concepts like encapsulation, the single responsibility principle, and cohesion. Breaching these principles might lead to complications, making extension, reuse, or maintenance challenging.
+
+In essence, the App CRD will have the DB CRD as its controller. Let's say, for instance, that the application requires a Deployment and Service to run. In this example, the App’s Controller will cater to these needs. Similarly, the DB’s controller will manage the business logic of its items.
```

Comment:
> In essence, the App CRD will have the DB CRD as its controller.

not sure I am following here, can you elaborate ?

Reply:
changed thx